### PR TITLE
Update to kubernetes-1.15.4

### PR DESCRIPTION
**Pull request by sslavic**
Towards giantswarm/giantswarm#6302

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -30,11 +30,11 @@
   name = "github.com/cenkalti/backoff"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be reverted once https://github.com/giantswarm/e2e-harness/pull/209 gets merged.

**Review by tomahawk28 — APPROVED** [review]

File: DCO [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
```
Comment (rossf7) [inline DCO]: Not a big thing but we usually do separate commit for DCO. Ping is fine doesn't need an approval.
